e3d/e3d_image.erl

```diff
@@ -167,76 +167,51 @@ bytes_pp(#e3d_image{bytes_pp = Bpp}) ->
 %% Desc: Filter and build a normalmap from a heightmap.
 %%       assumes the heightmap is greyscale.
 height2normal(Image, Scale, GenMipMap) ->
-    NM  = height2normal(Image, Scale),
+    NM  = height2normal(Image, [Scale,1.0,1.0]),
```

Comment:
Do not use a list here, lists are for data of arbitrary length. 
Use a tuple, or a proplists [{scale,S}, {invert_x, true|false}, {invert_y, true|false}
and if invert is a invert it should be booleans, i.e. true or false.

Then you can pre-multiply the Scale to {ScaleX, ScaleY} to have two less multiplication in the loop.

Otherwise the bump-maps with height (grayscale) works much better now.